css/index.css

```diff
@@ -112,6 +150,149 @@ p {
     justify-content: center;
 }
 
+.middle-content .box1 {
```

Comment:
You can actually condense this by using one class for general box styles, and having these box1, box2, etc classes only change the color